### DISCUSSION TITLE
[Stundenplan] Stundenplan Schüler

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#14] [User Stories]
### User Story
Als Schüler will ich, wenn ich den Stundenplan aufrufe, meinen Stundenplan sehen, damit ich auf einen Blick meine Stunden für die Woche sehen kann.
### Beschreibung:
Es soll sowohl eine Seite als auch ein kleines Fenster auf der Startseite geben, welches den Stundenplan anzeigt. Dabei soll nur der persönliche Stundenplan des jeweiligen Schülers sichtbar sein und nicht alle Stundenpläne.
Im Stundenplan soll erkennbar sein, wann der jeweilige Schüler welches Fach hat und welcher Lehrer die Stunde in welchem Raum hält. 

### Akzeptanz-Kriterien:

- [ ] Veränderungen am Stundenplan der Klasse werden dem Schüler direkt angezeigt 
- [ ] Nur der Stundenplan der Klasse des Schülers wird dem Schüler angezeigt

### Issues
Frontend: 
- #114 
- #295 
- #296

Backend: 
- #115

**Comment by laralngn** [comment]
Was meinst du hiermit: "nicht alle Stundenpläne, diese sollen im direkt auf der Webseite angezeigt werden mit den Stunden die sich eventuell geändert haben oder ausfällen."

**Reply by Kronprinz03** [reply]
Äh ist das noch aktuell da ich die Stelle nicht finde von der du mich zitierst meinst du "wenn ich den Stundenplan aufrufe direkt meinen eigenen Stundenplan sehen"?

**Reply by laralngn** [reply]
ja hatte die rausgenommen, aber das stand da vorher. Wusste nur nicht ob ich das wieder hinzufügen soll, weil ich nicht wusste was damit gemeint ist. Es ging irgendwie darum, dass halt nur der richtige Plan für den User angezeigt wird und dann kam das Zitat.

**Reply by tilwbr** [reply]
geht ja irgendwie darum, dass nur der korrekte stundenplan angezeigt wird, für mich ist das aber selbstverständlich denke ich
